translations/site.import.local.yml

```diff
@@ -1,16 +1,16 @@
-description: 'Import/Configure an existing local Drupal project'
-help: 'The <info>site:import:local</info> does create the yaml configuration file for a local existing site.'
+description: 'अस्तित्वातील स्थानिक Drupal प्रोजेक्ट आयात/कॉन्फिगर करा.'
+help: '<info>site:import:local</info> स्थानिक विद्यमान साइटसाठी yaml व्यूहरचना फाइल तयार करते.'
 arguments:
-    directory: 'Existing Drupal root directory'
-    name: 'Name that will be used to generate the site config'
+    directory: 'विद्यमान Drupal रूट निर्देशिका.'
+    name: 'साइट कॉन्फिगर तयार करण्यासाठी वापरला जाणार असलेले नाव.'
 options:
-    environment: 'Name of the environment that is going to be imported'
+    environment: 'पर्यावरणाचे नाव जे आयात केले जाणार आहे.'
 messages:
-    imported: 'The site has been imported successfully. You might want to run `drupal site:debug` to confirm that everything went well.'
-    error-missing: 'The directory "%s" doesn''t exist.'
-    error-not-drupal: 'This is not a valid drupal root: "%s"'
-    error-writing: 'An error accurred while trying to write the config file: "%s"'
+    imported: 'साइट यशस्वीरित्या आयात केली गेली आहे. आपण सर्वकाही चांगले केले याची पुष्टी करण्यासाठी `Drupal site:debug` चालवायचे असेल.'
```

Comment:
remove the capital drupal , the correct is drupal site:debug no Drupal site:debug